client/extensions/woocommerce/lib/order-status/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Lists of statuses in each group, waiting for payment, waiting for
+ * fulfillment, and finished orders.
+ */
+export const statusWaitingPayment = [
+	'on-hold',
+	'pending',
+];
+export const statusWaitingFulfillment = [
+	'processing',
+];
+export const statusFinished = [
+	'cancelled',
+	'completed',
+	'failed',
+	'refunded',
+];
+
+/**
+ * Get a list of order statuses for display (including a translated label)
+ *
+ * @return {Array} List of objects {name,value} for each status
+ */
+export function getOrderStatusList() {
+	return [ {
+		value: 'pending',
+		name: translate( 'Pending payment' ),
+	}, {
+		value: 'processing',
+		name: translate( 'Processing' ),
+	}, {
+		value: 'on-hold',
+		name: translate( 'On Hold' ),
+	}, {
+		value: 'completed',
+		name: translate( 'Completed' ),
+	}, {
+		value: 'cancelled',
+		name: translate( 'Cancelled' ),
+	}, {
+		value: 'refunded',
+		name: translate( 'Refunded' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store-orders-filters)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).